monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+# Triptych: Un Nouvel Algorithme Protégeant les Données des Utilisateurs de Monero
+
+**“Monero obfusque l'émetteur, le destinataire, et le montant des transactions. Triptych est une amélioration pour Monero qui rend son utilisation plus rapide, plus légère et plus privée.” - 1e septembre, 2020**
+
+Monero est conçu pour obfusquer les valeurs (émetteur, destinataire et montants) de chaque transaction avec des techniques et des technologies évolutives. Il s’agit de présenter ici Triptych, une technologie innovante permettant l’optimisation de la fonction d'obfuscation de l’émetteur, celui de l’usage de la chaine de blocs et une rapidité d’exécution supérieure.
```

Comment:
transaction avec des techniques et des technologies évolutives >>>   transaction avec des techniques et des technologies complexes et en constante évolution. 

Reason: I think the original text use 'evolving' as 'that can be improved constantly'. In other words, it is more somewhat synonym of 'continuously updated'. In the proposed French translation, this notion was lost.

Reply:
Integrated.